nbextensions/bulkImportCell/tabs/xsvGenerator.js

```diff
@@ -142,40 +142,50 @@ define(['StagingServiceClient', 'common/html', 'common/runtime', 'common/ui', 'u
 
         async run() {
             const body = this.renderLayout();
+            const formValues = {};
             const modalArgs = {
                 title: 'Create CSV Template',
                 body,
                 okLabel: 'Generate template!',
-                doThisFirst: (mdn) => {
-                    const submit = mdn.querySelector('[data-element="ok"]');
+                onConfirm: () => {
                     // TODO: the form should monitor its state of validity and
                     // only allow the query to be submitted when the form is valid
-                    submit.addEventListener('click', this.runRequest.bind(this));
```

Comment:
this turned out not to be a robust way to do it -- the `submit` element already has a click handler on it, and adding this one, which triggers another modal, was causing intermittent buggy behaviour.